docs/src/man/comparisons.md

```diff
@@ -0,0 +1,27 @@
+The table compares the main functions of DataFrames with R dplyr and Stata
```

Comment:
which versions of R/dplyr and Stata? If we do it also Pandas should be added I think.

Reply:
also later I would add DataFramesMeta.jl (when @pdeffebach is done with the work he is concentrating on now)

Reply:
also add that `df` is a data frame here.

Reply:
I do not know Panda unfortunately. Maybe someone could add it after?